addons/im_livechat/__manifest__.py

```diff
@@ -163,6 +163,7 @@
             ('include', 'im_livechat.assets_embed_external'),
             ('remove', 'im_livechat/static/src/embed/external/boot.js'),
             ('remove', 'mail/static/src/discuss/core/web/discuss_core_common_service_patch.js'),
+            'html_editor/static/tests/_helpers/**/*',
```

Comment:
Why is it necessary in this PR? I don't see any livechat tests being changed to make use of the new helpers.

Reply:
`mail_test_helpers` imported `html_editot_helpers` and so livechat uses `mail_test_helpers`